pyptex/__init__.py

```diff
@@ -565,35 +585,28 @@ def pyptexmain(argv=None):
     """
     argv = argv or sys.argv
     dopdb = True
-    try:
-        k = argv.index('--pdb=no')
-        argv.pop(k)
+    with suppress(ValueError):
+        argv.remove('--pdb=no')
         dopdb = False
-    except:
-        pass
-    try:
-        k = argv.index('--pdb=yes')
-        argv.pop(k)
+    with suppress(ValueError):
+        argv.remove('--pdb=yes')
         dopdb = True
-    except:
-        pass
-    if len(argv)<2:
-        print("Usage: pyptex <filename.tex> ...")
+    if len(argv) < 2:
+        print('Usage: pyptex <filename.tex> ...')
         sys.exit(1)
     try:
         # logging inspired by Jacob Gabrielson on Stackoverflow
-        tee = subprocess.Popen(["tee", os.path.splitext(argv[1])[0]+".pyplog"], stdin=subprocess.PIPE)
+        tee = subprocess.Popen(['tee', f'{os.path.splitext(argv[1])[0]}.pyplog'], stdin=subprocess.PIPE)
         os.dup2(tee.stdin.fileno(), sys.stdout.fileno())
         os.dup2(tee.stdin.fileno(), sys.stderr.fileno())
-        pyp = pyptex(argv[1],argv[2:],
-            latexcommand=r"{latex} {pyptexfilename} && (test ! -f {bibfilename} || bibtex {auxfilename})"
-            )
-    except:
+        pyp = pyptex(argv[1], argv[2:],
+            latexcommand=r'{latex} {pyptexfilename} && (test ! -f {bibfilename} || bibtex {auxfilename})')
```

Comment:
Eyes are watering again bc github is confused, just whitespace?

Reply:
Yup, just whitespace again.